README.md

```diff
@@ -151,3 +152,17 @@ SUBCOMMANDS:
 ```
 
 [cargo]: https://doc.rust-lang.org/cargo/getting-started/installation.html
```

Comment:
Here we have a link list with only one entry. Let's keep them on the bottom of the readme file, so moving below the new inserted subsections 'See also' and 'About'.
Adding the most frequent used links could be added:
```
[pallet-move]: https://github.com/eigerco/pallet-move
```